### PR TITLE
[IMP] Adding another PYTHONPATH in zshrc

### PR DESCRIPTION
**Pull request by JesusZapata**
Referring to https://github.com/Vauxoo/docker-odoo-image/pull/174#issuecomment-274151848

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 
Copy configuration from bashrc to zshrc!

**Comment by moylop260** [comment]
This matter is with @osvalr man, please

**Comment by JesusZapata** [comment]
@osvalr 
What do you think about this PR?
It is referring to you comment https://github.com/Vauxoo/docker-odoo-image/pull/174#issuecomment-274139587

File: odoo-shippable/scripts/build-image.sh [hunk 1]
```diff
@@ -203,6 +203,10 @@ cat >> /etc/bash.bashrc << EOF
 PYTHONPATH=${PYTHONPATH}:/.repo_requirements/odoo
 EOF
 
+cat >> /etc/zsh/zshrc << EOF
```
Comment (osvalr) [inline odoo-shippable/scripts/build-image.sh]: @moylop260 @JesusZapata could we please use `/root/.zshrc` instead.. 
I've tested, please see the following [asciinema](https://asciinema.org/a/auul5zgm5l0u1lqb9gcntnnqs)

**Comment by osvalr** [comment]
thank you @JesusZapata

File: odoo-shippable/scripts/build-image.sh (hunk 1 above)
Comment (JesusZapata) [inline odoo-shippable/scripts/build-image.sh]: @moylop260 @osvalr 
Done, I have used `~/.zshrc`

**Pull request merged**